app/views/layouts/application.html.slim

```diff
@@ -21,3 +21,5 @@ html
                 i.fa.fa-sign-out
     .container
       = yield
+    div id="footer"
```

Comment:
in slim, we dont have to specify something to be a div. this line can be written as `#footer` slim will assume you want to make a div that has an id of 'footer'